add-on/chrome/bootstrap.js

```diff
@@ -1383,6 +1383,15 @@ function loadDefaultPrefs() {
  * Called when the add-on is started, e.g. when installed or when Firefox starts.
  */
 function startup(data) {
+  // Anything later than FF 49 we don't support, so don't try to display the
+  // button. Later this will be to uninstall the add-on itself.
+  // Note: this check exists in shutdown() as well.
+  Cu.reportError(Services.appinfo.version);
```

Comment:
Is reportError the right thing to use here? Should there be a more detailed message here so people know where this message comes from?